src/main/java/frc/robot/RobotContainer.java

```diff
@@ -234,9 +238,11 @@ private void configureBindings() {
     // Trap Elv Intaking
     if (Constants.enabledSubsystems.elvEnabled) {
       OI.getButton(OI.Driver.groundIntakeButton)
-          .whileTrue(trapElvSubsystem.intakeGround().onlyWhile(trapElvSubsystem.getGroundBreak()));
+          .whileTrue(
+              trapElvSubsystem.intakeGround().onlyWhile(SensorManager.getGroundBreakBoolInverse()));
       OI.getButton(OI.Driver.sourceIntakeButton)
-          .whileTrue(trapElvSubsystem.intakeSource().onlyWhile(trapElvSubsystem.getSourceBreak()));
+          .whileTrue(
+              trapElvSubsystem.intakeSource().onlyWhile(SensorManager.getSourceBreakBoolInverse()));
```

Comment:
I think that sourcebreak and groundbreak are actually the same sensor.

If I understand the naming.